docker/test/Dockerfile

```diff
@@ -36,3 +30,11 @@ ADD templates /cluster-broccoli-dist/templates
 RUN mkdir /cluster-broccoli-dist/instances
 RUN ln -s /cluster-broccoli-dist/bin/cluster-broccoli /usr/bin/cluster-broccoli
 
+# put nomad
+ARG nomad="0.4.0"
```

Comment:
0.8.0

Reply:
The tests will break if I do this because a lot of them depend on `JSON` responses that have changed over the versions. I will take this up as a separate PR.